### DISCUSSION TITLE
CarpetSensorModeControlCapability through MQTT

### DISCUSSION THREAD

**Discussion by Reinachan** [Hypfer/Valetudo#1907] [Q&A / Support]
Would adding CarpetSensorModeControlCapability to MQTT be something you'd consider? I'd like to use it in automations so I can avoid the carpet in the livingroom as the robot gets stuck in it, but elsewhere I want it to vacuum carpets.

My current workaround is to set up a REST call with the http integration for Home Assistant, however, I'd prefer having it properly integrated with MQTT.

Edit: I figured you prefer only getting bug reports as issues, so I didn't post this as a feature request there. If you want me to move this to a feature request issue, I can do that

**Comment by s1mpleman** [comment]
Came here to add my support for this. 

Would love to be able to toggle this on/off as when it's on currently and my vacuum runs at night we can hear it. I didn't realise there was an API, will use that for now.
